samples/ClassroomLabs/Module/Library/HowTo.md

```diff
@@ -0,0 +1,152 @@
+# Az.AzureLabs Tutorial <!-- omit in TOC -->
+
+Az.AzureLabs is a PowerShell module to simplify the management of [Azure Lab services](https://azure.microsoft.com/en-in/services/lab-services/). It provides composable functions to create, query, update and delete lab accounts, labs, VMs and Images.
+
+- [Introduction](#introduction)
+- [Import the module](#import-the-module)
+- [Browse all the functions in the library](#browse-all-the-functions-in-the-library)
+- [Publish a new lab](#publish-a-new-lab)
+- [Query for lab accounts, labs and VMs](#query-for-lab-accounts-labs-and-vms)
+- [Manage Users](#manage-users)
+- [Set schedules](#set-schedules)
+- [Remove objects](#remove-objects)
+- [Give us feedback](#give-us-feedback)
+  
+## Introduction
+
+If you want a quick overview of all the features, see this [scenario file](Scenarios/AllFeatures.ps1).
+
+Many of the code snippets below might take a bit of time (minutes) to execute, especially the ones that create Azure resources.
+
+You can pass the `-verbose` parameter to see some action on your screen, otherwise the snippets succeed silently.
+
+## Import the module
+
+First you need to save the [`Az.AzureLabs.psm1`](Az.AzureLabs.psm1) file in a directory on disk.
+
+Then you need to open a Powershell console and import the module:
+
+```powershell
+cd DIRECTORY-WITH-MODULE
+import-module ./Az.AzureLabs.psm1
+```
+
+## Browse all the functions in the library
+
+As a quick way to see which DevTestLabs capabilities are supported, type the following command in the console.
+
+```powershell
+Get-Command -Module Az.AzureLabs
+```
+
+## Publish a new lab
+
+From now on, the tutorial assumes that you are signed in into Azure. If you are not, you can read about it [here](https://docs.microsoft.com/en-us/powershell/azure/authenticate-azureps?view=azps-2.1.0).
+
+First, let's create a new Resource Group to host your lab.
+
+```powershell
+New-AzureRmResourceGroup -Name DtlPS1 -Location "West Europe"
```

Comment:
I have used `New-AzureRmResourceGroup` throughout because of the desire of this to be run from old PS1 versions which customers might have in their shops. It is the same for the DTL library. It can be run from newer PS1 (i.e. Core) by enabling aliases. I am not sure that's the right thing overall, but it's what we have done for the DTL library.

Reply:
My comment stemmed from the fact that elsewhere you use `New-AzResourceGroup`. So, for consistency, I would be inclined to use the same throughout.